orbital-mechanics/Lecture7/Lecture7.ipynb

```diff
@@ -1,386 +1,425 @@
 {
```

Comment:
<p>Please put both equations for a_2 and e_2 as full equations. Don't worry about creating labels for the equations. In fact, let's just avoid equation numbers for these two and just put each equation within $ as below. </p><p> </p><p>$$</p><p>equation</p><p>$$</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>